external/fv3kube/fv3kube/config.py

```diff
@@ -76,11 +82,19 @@ def get_base_fv3config(version_key: Optional[str] = None) -> FV3Config:
     return base_yaml
 
 
+class RestartCategoriesConfig(Protocol):
```

Comment:
I can't see any need for multiple implementations of "RestartCategoriesConfig" so I don't see the need for this Protocol layer. Maybe just define the dataclass here instead of using a protocol?

Reply:
Such a dataclass gets usage in both fv3kube and runtime, so yeah if it's imported from fv3kube into runtime then it only needs to be defined once (but is maybe less discoverable in runtime). I'll go with that.